sentry/src/main/java/io/sentry/IScopeObserver.java

```diff
@@ -0,0 +1,9 @@
+package io.sentry;
+
+import io.sentry.protocol.User;
+
+public interface IScopeObserver {
```

Comment:
M: we'd need to implement the other fields too, like `extra`, `fingerprint`... and also to remove them, like `removeTag`...

Reply:
The customer only asked about user,tag and breadcrumbs but yeah, for completion we'd have to add a bunch more stuff

Reply:
or to document pretty well what we sync and what we don't, I'm fine either way.

Reply:
Could be an option. Specially if it'll require lots of JNI calls to sync stuff.

Reply:
tags start with the min. necessary to unblock folks and see how it goes, we can implement the others later then.
starting with tags, extras, user, and breadcrumbs.